test/unittest/Database/DatabaseDumpTests.cpp

```diff
@@ -454,6 +454,19 @@ TEST(database, dump_and_clear_database)
                 ASSERT_FALSE(it.second->data.rtps_bytes_sent.empty());
                 ASSERT_FALSE(it.second->data.rtps_packets_lost.empty());
                 ASSERT_FALSE(it.second->data.rtps_bytes_lost.empty());
+
+                ASSERT_FALSE(it.second->data.last_reported_rtps_packets_sent_count.empty());
+                ASSERT_FALSE(it.second->data.last_reported_rtps_bytes_sent_count.empty());
+                ASSERT_FALSE(it.second->data.last_reported_rtps_packets_lost_count.empty());
+                ASSERT_FALSE(it.second->data.last_reported_rtps_bytes_lost_count.empty());
+                ASSERT_FALSE(it.second->data.last_reported_pdp_packets.kind == DataKind::INVALID &&
+                        it.second->data.last_reported_pdp_packets.src_ts ==
+                        std::chrono::system_clock::time_point() &&
+                        it.second->data.last_reported_edp_packets.count == 0);
```

Comment:
```suggestion
                ASSERT_FALSE(it.second->data.last_reported_pdp_packets.kind == DataKind::INVALID ||
                        it.second->data.last_reported_pdp_packets.src_ts ==
                        std::chrono::system_clock::time_point() ||
                        it.second->data.last_reported_edp_packets.count == 0);
```
I believe none of these values should be the default one.
The same with the rest of checks

Reply:
We discussed it and came to the conclusion of leave it at it is.